Sources/MongoKitten/AggregateStage.swift

```diff
@@ -2,31 +2,34 @@ import MongoClient
 
 public struct AggregateBuilderStage {
     public internal(set) var stages: [Document]
+    public internal(set) var minimalVersionRequired: WireVersion?
     
-    public init(document: Document) {
+    public init(document: Document, _ minimalVersionRequired: WireVersion? = nil) {
```

Comment:
I'm not a fan of omitting the parameter's label. If the variable name is too long, maybe just use the parameter & label `minimumVersion`